fetch_threads.py

```
@@ -0,0 +1,20 @@
+import json
+import subprocess
+import re
+
+res = subprocess.run(['gh', 'api', 'graphql', '-f', 'query=query { repository(owner: "dwojtaszek", name: "zipper") { pullRequest(number: 512) { reviewThreads(first: 100) { nodes { id isResolved comments(first: 5) { nodes { body path line } } } } } } }'], capture_output=True, text=True)
```

Ruff (0.15.17)

[error] Starting a process with a partial executable path

(S607)